erpnext/projects/doctype/project/project.py

```diff
@@ -176,6 +176,13 @@ def update_purchase_costing(self):
 			from `tabPurchase Invoice Item` where project = %s and docstatus=1""", self.name)
 
 		self.total_purchase_cost = total_purchase_cost and total_purchase_cost[0][0] or 0
+		
+	def update_sales_costing(self):
+		total_sales_cost = frappe.db.sql("""select sum(total)
```

Comment:
You're right, I was thinking about how the billing hours don't show taxes, but those are further malleable unlike SO/SINV